docs/import-images.md

```diff
@@ -0,0 +1,74 @@
+
+
+## Pre-import images
+:::info Version Gate
+The pre-importing of images while K3s is running feature is available as of January 2025 releases: v1.32.0+k3s1, v1.31.5+k3s1, v1.30.9+k3s1, v1.29.13+k3s1. Before that, K3s pre-imported the images only when booting.
+:::
+
+Pre-importing images onto the node is essential if you configure Kubernetes' `imagePullPolicy` as `Never`, for example, for security reasons. Another reason to do this is to reduce the time it takes for your K3s nodes to spin up.
+
+K3s includes two mechanisms to pre-import images into the containerd image store:
+
+<Tabs groupId = "import images" queryString>
+<TabItem value="Online image importing" default>
+
+Users can trigger the pulling of images into the containerd image store by adding a `.txt` file containing the image paths to the `/var/lib/rancher/k3s/agent/images` directory while K3s is running. A K3s controller will detect this file and trigger the image pull from an upstream (default) or [private registry](installation/private-registry.md).
+
+For example:
+
+```bash
+mkdir /var/lib/rancher/k3s/agent/images
+cp example.txt /var/lib/rancher/k3s/agent/images
+```
+
+Where `example.txt` contains:
+
+```
+docker.io/library/redis:latest
+docker.io/library/mysql:latest
+```
+
+After a few seconds, the `redis` and the `mysql` images will be available in the containerd image store of the node. 
+
+Use `sudo k3s ctr images list` to query the containerd image store.
+
+</TabItem>
+<TabItem value="Offline image importing">
+
+Users can import images offline to containerd by adding image tarballs in the directory `/var/lib/rancher/k3s/agent/images` while K3s is running. A K3s controller will detect the image tarball, extract the images and store them in the containerd image store.
+
+For example:
+
+```bash
+mkdir /var/lib/rancher/k3s/agent/images
+curl  https://github.com/k3s-io/k3s/releases/download/v1.33.1%2Bk3s1/k3s-airgap-images-amd64.tar.zst -O  /var/lib/rancher/k3s/agent/images/k3s-airgap-images-amd64.tar.zst
+```
+
+After a few seconds, the images included in the image tarball will be available in the containerd image store of the node. 
+
+Use `sudo k3s ctr images list` to query the containerd image store.
+
+:::warning
+When using the offline image importing for Airgap, tarballs should be present before K3s starts. Please follow the [airgap install documentation](installation/airgap.md) for detailed information
```

Comment:
Maybe say why this is the case? Its fine if users want to add them later, just the pods will be stuck in ImagePullBackoff until they make the images available.